get_http_test.go

```diff
@@ -207,6 +209,22 @@ func TestHttpGetter_authNetrc(t *testing.T) {
 	}
 }
 
+func TestHttpGetter_GetByteRange(t *testing.T) {
```

Comment:
Happy to see tests 😄 I'd turn this into a table test, something of this structure:

```go
cases := []struct {
    Input string // example: "5555-66666"
    Expected []string
    Err bool
}
```

Then we can test all the edges and all the error branches in the func.